### ISSUE TITLE
Change the local frame of reference

### ISSUE THREAD

**Issue by MrPrietix** [PX4/PX4-Autopilot#19460]
Hello

Is there anyway to change the latitude, longitude and altitude of the local frame of reference that px4 is using?

Thanks in advance

MrPrietix

**Comment by bresch** [comment]
Yes, it is. I'll let you check here: https://github.com/PX4/PX4-Autopilot/pull/16544
